src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -282,6 +401,13 @@ def _sanitize_parameters(
             postprocess_params["decoder_kwargs"] = decoder_kwargs
         if return_timestamps is not None:
             postprocess_params["return_timestamps"] = return_timestamps
+            if self.model.config.__class__.__name__ == "WhisperConfig":
```

Comment:
We cannot use `self.type == "seq2seq_whisper"` here since this is called before `self.type` exists during `__init__`.

Reply:
Same, let's check `self.model.config.model_type` here.